taprpc/taprootassets.proto

```diff
@@ -611,6 +615,25 @@ message ListTransfersResponse {
     repeated AssetTransfer transfers = 1;
 }
 
+message SetPendingTransferProofCourierAddrRequest {
+    // The new proof courier address to use for the target transfer output proof
+    // delivery.
+    string new_proof_courier_addr = 1;
```

Comment:
Maybe to make it more clear we can only update this field (for now?), we could put that into a new sub-message.

So something like this:

```
message UpdateTransferOutput {
    // The hexadecimal encoded transaction ID (txid) of the anchor transaction
    // for the target transfer.
    string anchor_txid = 1;

    // Script public key of the target transfer output.
    bytes transfer_output_script_pub_key = 2;

    // The position of the output in the transfer output list.
    uint64 transfer_output_position = 3;

    TransferOutputModify modify_params = 4;
}

message TransferOutputModify {
    string proof_courier_addr = 1;
```

Reply:
With the current definition I think this is the only reasonable field to change rn, but I agree that using a sub-message is a worthwhile change.